node-repository/src/main/java/com/yahoo/vespa/hosted/provision/os/OsVersions.java

```diff
@@ -96,11 +103,33 @@ public void setTarget(NodeType nodeType, Version newTarget, boolean force) {
                                                    + oldTarget.get().version());
             }
 
-            osVersions.put(nodeType, new OsVersion(newTarget, true));
+            osVersions.put(nodeType, new OsVersion(newTarget, false));
```

Comment:
Copy active state from `oldTarget` if present?

Reply:
Setting the same version is a no-op (checked earlier in this method), but I can add a test for this case.

Reply:
I meant to copy the active state, not the version? I.e. if we were upgrading to X and it was `active`, then upgrading to `y` should also be `active`? No need to set to false and wait for the next iteration of the maintainer to set back to `active`?

Reply:
```suggestion
osVersions.put(nodeType, new OsVersion(newTarget, oldTarget.map(OsVersion::active).orElse(false)));
```

Reply:
True, I realised that posting my comment. We can do that, although I don't think it matters much in practice.

Reply:
Actually, let's leave it as-is. I think it's more clean to let `OsUpgradeActivator` be the sole thing that activates versions (as stated in javadoc).